test/distributed/org/apache/cassandra/distributed/action/GossipHelper.java

```diff
@@ -228,7 +229,7 @@ public void accept(IInvokableInstance pullTo)
                 InetAddressAndPort endpoint = toCassandraInetAddressAndPort(pullFrom);
                 EndpointState state = Gossiper.instance.getEndpointStateForEndpoint(endpoint);
                 Gossiper.instance.doOnChangeNotifications(endpoint, ApplicationState.SCHEMA, state.getApplicationState(ApplicationState.SCHEMA));
-                Schema.instance.waitUntilReady(Duration.ofSeconds(10));
+                assertTrue("schema is ready", Schema.instance.waitUntilReady(Duration.ofSeconds(10)));
```

Comment:
you are a good person here, but not in 4.0?  why the hate of 4.0!?!?!!

Reply:
Puzzler. It was there, I'll put it back.